crates/ecolor/src/color32.rs

```diff
@@ -235,4 +235,16 @@ impl Color32 {
             a as f32 / 255.0,
         ]
     }
+
+    /// Lerp this color towards `other` by `t` in gamma space.
```

Comment:
Docstring kindly provided by Copilot. I'm still largely confused by the linear vs. gamma distinction.